src/components/Chip.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Box as BoxPrimitive, BoxProps as BoxPrimitiveProps } from 'mdlz-prmtz';
+import { variant, Prop } from '@modulz/radix-system';
+import { transparentize } from 'polished';
+import {theme} from '../theme';
+
+type ChipProps = BoxPrimitiveProps & {
+  variant?: Prop<'gray' | 'primary' | 'blue' | 'green' | 'purple' | 'orange'>;
+  size?: Prop<0 | 1 | 2>;
+  truncate?: Prop<boolean>;
+};
+
+export const Chip = React.forwardRef<HTMLDivElement, ChipProps>((props, ref) => (
+  <BoxPrimitive
+    {...props}
+    ref={ref}
+    css={[
+      css({
+        fontSize: 'inherit',
+        fontWeight: 600,
+        margin: '4px',
+        alignSelf: 'flex-start',
+        display: 'inline-block',
+        minWidth: '20px',
+        textAlign: 'center',
+        borderRadius: '12px',
+      }),
+      variant({
+        variant: {
+          gray: {
+            color: 'grays.7',
+            backgroundColor: 'grays.2',
+          },
+          primary: {
+            color: 'primary',
+            backgroundColor: transparentize(0.85, theme.colors.primary),
+          },
+          blue: {
+            color: 'blue',
+            backgroundColor: 'blues.2',
```

Comment:
I think we should be consistant and use transparentize on the based color for all chip colors. (ex: `backgroundColor: transparentize(0.85, theme.colors.blue)/transparentize(0.85, theme.colors.green)/transparentize(0.85, theme.colors.orange),`

Reply:
I found that `setLightness` is better for this situation since we may want to place a chip into a colored background and if the color has some transparency, the background may affect the chip color. 
So I replaced `transparentize` for `setLightness` and used it consistently for all variants. Thank you :slightly_smiling_face: